README.md

```diff
@@ -20,11 +20,100 @@ This repository contains the core Mobius framework and add-ons for common develo
 | 🛠 Xcode    | 10.1+       |
 | 🐦 Language | Swift 4.2  |
 
-## Usage
+## Usage in your project
 Pull in Mobius.swift as a dependency. Either as a submodule or using [Carthage](https://github.com/Carthage/Carthage).
 
 Build the project and link with the frameworks.
 
+## Mobius in action - Building a Counter
+
+The goal of Mobius is to you better control over your application state. You can think of your state as a snapshot of all the current values of the variables in your application.In Mobius, we take encapsulate all of the state in a data-structure which we call the *Model*.
+
+The *Model* can be represented by whatever type you like. Since we're building a counter, we'll be able to encapsulate all of our state in an `Int`:
+
+```swift
+typealias MyModel = Int
```

Comment:
Can we rename these entities to something like `CounterModel` and `CounterEvent` instead of `My`?